apollo-portal/src/main/resources/static/scripts/controller/IndexController.js

```diff
@@ -15,21 +15,142 @@
  *
  */
 index_module.controller('IndexController', ['$scope', '$window', '$translate', 'toastr', 'AppUtil', 'AppService',
-    'UserService', 'FavoriteService',
-    IndexController]);
+    'UserService', 'FavoriteService', 'NamespaceService',
+    IndexController]
+)
+.directive('inputSelect', function() {
+    return {
+        link: function(scope, element, attr) {
+            element.on('click', function(evt) {
+                evt.target.parentElement.previousElementSibling.value = evt.target.textContent;
+                scope.inValue = evt.target.textContent;
+                if(evt.target.parentElement.getElementsByClassName('item-bg').length){
+                    angular.element(evt.target.parentElement.getElementsByClassName('item-bg')[0]).removeClass('item-bg');
+                }
+                angular.element(evt.target).addClass('item-bg');
+                angular.element(evt.target.parentElement).addClass('hidden-cls');
+                return true;
+            });
+        }
+    };
+})
+.directive('inputSearch', function() {
```

Comment:
I suggest we use the existing select2 component as the search box, just as the top search bar.
You may reference the implementation in directive.js

https://github.com/ctripcorp/apollo/blob/772d2fa862b6622bbeb31b50437680887a35e749/apollo-portal/src/main/resources/static/scripts/directive/directive.js#L37-L87

Reply:
@youabcd Are you still working on this? I think we could merge this pr when this issue is resolved